### DISCUSSION TITLE
Welcome to the FeTS 2021 Challenge!

### DISCUSSION THREAD

**Discussion by sbakas** [FETS-AI/Challenge#3] [General]
### The 1st Computational Competition on Federated Learning.

We would like to use GitHub Discussions as the place to **ask questions, share ideas, and engage** with colleagues interested in FeTS 2021, as well as to provide further informational material about the Challenge as it progresses. Please post any particular question you might have in the "Q&A" section here.

The FeTS 2021 challenge focuses on federated learning for the task of brain tumor segmentation. For those of you who previously participated in the BraTS challenge, it is worth noting that the FeTS Challenge will be different on:
1. the MRI dataset being provided together with their source information.
2. your submission for evaluation requires your code instead of your predicted segmentations, as we would need to run your method in unseen data that reside on collaborating institutions across the globe.

Instructions relating to data and participation details for each task are provided in [the FeTS challenge homepage](https://www.med.upenn.edu/cbica/fets/miccai2021/). We intend to provide additional information on implementation and requirement specifications for each particular task, as well as code examples and instructional videos to help you get started. 

Thanks again for your interest in this year’s FeTS Challenge, and we look forward to your participation.

Best wishes,

The FeTS 2021 organizing committee.

**Comment by hui-po-wang** [comment]
Hi @sbakas,

Thanks for organizing this exciting challenge. Is there any news on implementation and requirement specifications? If we'd like to start now, is there any available codebase we could try out?

**Reply by sarthakpati** [reply]
Thank you for your interest, @a514514772. You can find more details about Task 1 [here](https://github.com/FETS-AI/Challenge/tree/main/Task_1).

Best wishes,

The FeTS 2021 organizing committee

**Reply by hui-po-wang** [reply]
Thanks for the reply!

I still have few questions. 

1. What functions can we modify and submit exactly? I knew you already provided few examples in the jupyter notebook. Are they all we can modify? How about other parameters, such as rounds_to_train?
2. In the official document, one evaluation metric is sent/received bytes, but I didn't see how it's implemented in the jupyter notebook. Would you provide the details of how you measure it and let us measure it on our side as well?
3. Due to some reasons, our cluster has a wall time limit of 24h. How do I do checkpointing on your framework?

many thanks.

**Reply by sarthakpati** [reply]
This will be replied to in the separate discussion #14. All the best!
